docs/source/background.md

```diff
@@ -45,6 +45,46 @@ only their names, it should use defaults defined in {class}`arviz_base.rcParams`
 As the array facing class API is defined and should be common between submodules,
 that means that this class can very often be limited to an instance of the base array facing class.
 
+#### On `dim`, `dims` and `sample_dims`
+At one point in time, `dim` and `dims` coexisted as equivalent arguments,
+some functions had one, other the other but their behaviour was exactly the same.
+Then the xarray developers decided to enforce consistency and use `dim` everywhere.
+We decided to have `arviz-stats` follow that convention.
+
+`sample_dims` is not another name for the same argument but a different concept altogether.
+Both in xarray and in ArviZ `sample_dims` are expected to be present in all variables
+of the input. Let's see an example.
+
+Suppose we have a {class}`xarray.Dataset` `ds` with multiple variables:
+
+* `mu (chain: 4, draw: 100)`
+* `theta (chain: 4, draw: 100, school: 8)`
+
+It is perfectly valid to do `ds.mean(dim=["chain", "school"])` even though "school" is not a dimension
```

Comment:
We may remove the text here to a more "visible" guide. For instance part of this is already covered in the new overview (for xarray interface). And we can move the rest either there or to the array interface.